devdocs-generator.ps1

```diff
@@ -0,0 +1,195 @@
+<#
+
+    .DESCRIPTION
+    This script generates markdown files for the development documentation based on the existing JSON files.
+    Create table of content.
+
+#>
+
+# Load the JSON files
+$tweaks = Get-Content -Path "config/tweaks.json" | ConvertFrom-Json
+$features = Get-Content -Path "config/feature.json" | ConvertFrom-Json
+
+# Create the output directories if they don't exist
+$tweaksOutputDir = "docs/dev/tweaks"
+$featuresOutputDir = "docs/dev/features"
+
+if (-Not (Test-Path -Path $tweaksOutputDir)) {
+    New-Item -ItemType Directory -Path $tweaksOutputDir | Out-Null
+}
+
+if (-Not (Test-Path -Path $featuresOutputDir)) {
+    New-Item -ItemType Directory -Path $featuresOutputDir | Out-Null
+}
+
+# Function to generate markdown files
+function Generate-MarkdownFiles($data, $outputDir, $jsonFilePath, $type) {
+    $tocEntries = @()
+
+    foreach ($itemName in $data.PSObject.Properties.Name) {
+        $itemDetails = $data.$itemName
+        $category = $itemDetails.Category -replace '[^a-zA-Z0-9]', '-' # Sanitize category name for directory
+        $categoryDir = "$outputDir/$category"
+
+        # Create the category directory if it doesn't exist
+        if (-Not (Test-Path -Path $categoryDir)) {
+            New-Item -ItemType Directory -Path $categoryDir | Out-Null
+        }
+
+        $filename = "$categoryDir/$itemName.md"
```

Comment:
If possible, could it remove the `WPF<CategoryName>` prefix for file names? as currently it doesn't make sense to be having such long names when there're put in their appropriate category.

This's just a suggestion, and you can simply ignore it if you don't see benefits from it...

Reply:
will look into it, thanks :)

Reply:
What do you think about the solution?
gh-pages workflow should finish any moment..

Reply:
Looks good @MyDrift-user